foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,81 @@
-    *   Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
-
-5.  **Important note:** you want to `return` the results of this function call, _not_ `console.log()` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
-
-    ~~~javascript
-    function playRound(playerSelection, computerSelection) {
-      // your code here!
-    }
- 
-    const playerSelection = "rock";
-    const computerSelection = getComputerChoice();
-    console.log(playRound(playerSelection, computerSelection));
-    ~~~
-
-6.  Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-    *   You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
-    *   At this point you should be using `console.log()` to display the results of each round and the winner at the end.
-    *   Use `prompt()` to get input from the user. [Read the docs here if you need to.](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
-    *   Feel free to re-work your previous functions if you need to. Specifically, you might want to change the return value to something more useful.
-    *   Feel free to create more "helper" functions if you think it would be useful.
+Remember to commit early and often! To refresh your memory, check out the [commit messages lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages).
+
+#### Step 1: initial structure
+
+1. Create a new Git repository for your project.
+1. Create a blank HTML document with a script tag.
+
+It's best practice to link to an external JavaScript file inside this script tag. Using an external JavaScript file keeps your HTML file clean and organized.
+
+You don't have to write additional code in the HTML file. This game is played entirely on the console.
+
+#### Step 2: get computer choice function
+
+Your game will be played against the computer. Create a function named `getComputerChoice` that randomly returns 'Rock', 'Paper' or 'Scissors':
+
+1. Create a new function named `getComputerChoice`.
+1. Write the code to let `getComputerChoice` randomly `return` one of the following string values: "Rock", "Paper" or "Scissors". 
+   - **Hint**: The [Math.random](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) method returns a random number that's greater than or equal to 0 and less than 1. Think about how you can use this to conditionally return one of the multiple choices.
```

Comment:
```suggestion
   - **Hint**: The [`Math.random`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) method returns a random number that's greater than or equal to 0 and less than 1. Think about how you can use this to conditionally return one of the multiple choices. You may find the [`Math.floor`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor) method useful.
```

It could potentially be confusing if the learner doesn't know about the Math.floor method if they wanted to generate a random index.

Reply:
Technically, they shouldn't be using arrays as they haven't learned about it.

Reply:
> Technically, they shouldn't be using arrays as they haven't learned about them.

Oh yeah, on point. No need to add that link here then